sql/plsql/PlSqlParser.g4

```diff
@@ -4696,6 +4696,7 @@ merge_table_partition
 
 modify_table_partition
     : MODIFY PARTITION partition_name ((ADD | DROP) list_values_clause)? (ADD range_subpartition_desc)? (REBUILD? UNUSABLE LOCAL INDEXES)?
+    | MODIFY range_partitions
```

Comment:
Please extract common part:

```suggestion
    : MODIFY (PARTITION partition_name ((ADD | DROP) list_values_clause)? (ADD range_subpartition_desc)? (REBUILD? UNUSABLE LOCAL INDEXES)?
             | range_partitions
             )
```

Reply:
Thanks for the information, i've adjusted the rule and added an example. I guess amending commits is fine?